config/301-mongodbtarget.yaml

```diff
@@ -0,0 +1,214 @@
+# Copyright 2023 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: mongodbtargets.targets.triggermesh.io
+  labels:
+    knative.dev/crd-install: 'true'
+    triggermesh.io/crd-install: 'true'
+    duck.knative.dev/addressable: 'true'
+  annotations:
+    registry.knative.dev/eventTypes: |
+      [
+        {
+          "type": "io.triggermesh.mongodb.response",
+          "schema": "https://raw.githubusercontent.com/triggermesh/triggermesh/main/schemas/io.triggermesh.mongodb.response.json",
+          "description": "the gneric response to all requests"
+        }
+      ]
+    registry.triggermesh.io/acceptedEventTypes: |
+      [
+        { "type":  "io.triggermesh.mongodb.insert" },
+        { "type":  "io.triggermesh.mongodb.query.kv" },
+        { "type":  "io.triggermesh.mongodb.update" },
+        { "type": "*" }
```

Comment:
I think this (*) should be deleted since the only accepted event types are the others.

Reply:
It does accept arbitrary events :)